web/src/App.tsx

```diff
@@ -8,6 +8,8 @@ import AssetPage from "@reearth-cms/components/pages/Asset/Asset";
 import AssetListPage from "@reearth-cms/components/pages/Asset/AssetList";
 import DashboardPage from "@reearth-cms/components/pages/Dashboard";
 import MembersPage from "@reearth-cms/components/pages/Members";
+import MyIntegrationPage from "@reearth-cms/components/pages/MyIntegration";
```

Comment:
`MyIntegrationsPage`

Reply:
Fixed.

Reply:
Seems you missed the imported name change. It should be plural but still is MyIntegrationPage

Reply:
@KaWaite , done for all namings.